src/khoj/processor/conversation/prompts.py

```diff
@@ -18,6 +18,33 @@
 """.strip()
 )
 
+general_conversation_falcon = PromptTemplate.from_template(
+    """
+Using your general knowledge and our past conversations as context, answer the following question.
+### Instruct:\n{query}\n
```

Comment:
Given the prompt template is triple quoted shouldn't `{query}` just be put on the next line directly instead of using `\n`? Similar comment for the other prompt templates for falcon

```suggestion
### Instruct:
{query}

```

Reply:
Fair point. I thought these were literally the same, but perhaps it's not with string literals. In any case, I'll just standardize the formatting.